app/services/support_interface/send_initial_emails.rb

```diff
@@ -0,0 +1,15 @@
+module SupportInterface
+  class SendInitialEmails
```

Comment:
Please could this class have a more descriptive name mentioning UCAS?

Reply:
Renamed to `SendUCASMatchInitialEmails..` to keep it consistent with the email names e.g. `ucas_match_initial_email`, `ucas_match_initial_email_duplicate_applications` etc